src/components/common/dialogs/CustomDialog.js

```diff
@@ -110,7 +112,9 @@ const CustomDialog = ({
               // otherwise useContext(GlobalTogglesContext) will return undefined for
               // any custom dialog component (e.g. ExplanationDialog and other ones)
               <GlobalTogglesContext.Provider value={globalToggleState}>
-                <DialogContext.Provider value={dialogState}>{content}</DialogContext.Provider>
+                <GoodWalletContext.Provider value={goodWalletState}>
+                  <DialogContext.Provider value={dialogState}>{content}</DialogContext.Provider>
```

Comment:
doesn't it break anything ?

Reply:
No, it just passes GoodWalletContext to the Portal content